MREUnityRuntime/MREUnityRuntimeLib/PluginInterfaces/Behaviors/IPenBehavior.cs

```diff
@@ -7,7 +7,7 @@ namespace MixedRealityExtension.PluginInterfaces.Behaviors
 	/// <summary>
 	/// Interface that represents a pen behavior in the MRE runtime.
 	/// </summary>
-	public interface IPenBehavior : IToolBehavior<PenData>
+	public interface IPenBehavior : IToolBehavior
```

Comment:
Same: why does this exist?

Reply:
See answer in the button behavior interface

---
In reply to: [434211833](https://github.com/microsoft/mixed-reality-extension-unity/pull/160#discussion_r434211833) [](ancestors = 434211833)